youtube_dl/extractor/ccma.py

```diff
@@ -74,15 +85,25 @@ def _real_extract(self, url):
         title = informacio['titol']
         durada = informacio.get('durada', {})
         duration = int_or_none(durada.get('milisegons'), 1000) or parse_duration(durada.get('text'))
-        timestamp = parse_iso8601(informacio.get('data_emissio', {}).get('utc'))
+
+        # utc date is in format YYYY-DD-MM
+        data_utc = informacio.get('data_emissio', {}).get('utc')
+        try:
+            data_iso8601 = data_utc[:5] + data_utc[8:10] + '-' + data_utc[5:7] + data_utc[10:]
```

Comment:
use `datetime` class directly.

Reply:
Note this is not a standardized UTC timestamp... it is provided in format `YYYY-DD-MM`, thus this manual parsing was needed.

Reply:
that's why i think it's better to directly use python's datetime builtin module to parse the non standardized format.

Reply:
Sorry, but I don't follow...

I can only think of two options:
- Modifying `data_utc` as so that it is ISO-compliant and accepted as `datetime.datetime.fromisoformat(data_utc)`, but this is what is already being done...
- Parsing each segment as an `int` to call `datetime.datetime(year, month, day, hour=0, minute=0, second=0, microsecond=0...` constructor, but I see no advantage and only added complexity in comparison to what is currently being done.

Could you please give me another hint on what you have in mind?

Thanks :)